Automattic/wp-calypso
# Packages: Remove unused `@automattic/launch` package and launch store

## Proposed Changes

This PR removes the unused `automattic/launch` store and the unused `@automattic/launch` package.

Part of #74399. A follow-up to #73890.

## Testing Instructions

* Verify that the removed store is not in use.
* Verify the removed package is not in use.
* All checks should be green and Calypso should build well.
